indico/modules/events/contributions/util.py

```diff
@@ -72,3 +78,80 @@ def serialize_contribution_person_link(person_link):
     data['authorType'] = person_link.author_type.value
     data['isSubmitter'] = person_link.is_submitter
     return data
+
+
+class ContributionReporter(ReporterBase):
+    """Class performing reporting and filtering actions in the contribution
+    report
+    """
+    CSRF_ENABLED = True
+    REPORT_ID = 'contribution_report'
```

Comment:
i don't think we have a strong rule for it though... some class attributes are uppercase, others are lowercase